sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsAudience.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.ComponentModel;
+using Azure.Core;
+
+namespace Azure.AI.TextAnalytics
+{
+    /// <summary> Cloud audiences available for TextAnalytics. </summary>
+    public readonly partial struct TextAnalyticsAudience : IEquatable<TextAnalyticsAudience>
+    {
+        private readonly string _value;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="TextAnalyticsAudience"/> object.
+        /// </summary>
+        /// <param name="value">A string representing the Azure Cloud this service will access. Default to Azure Public Cloud. Read more at: https://azure.microsoft.com/en-us/overview/what-is-a-public-cloud/.</param>
```

Comment:
Make sure not to include the localized (en-us) link. Also perhaps we should link to a cognitive service specific doc here rather than general gov cloud information. Something like this? https://docs.microsoft.com/azure/azure-government/compare-azure-government-global-azure